src/nasher/utils/shared.nim

```diff
@@ -108,15 +108,16 @@ proc getNwnRootDir*: string =
     let settingsFile = getHomeDir() / "AppData" / "Roaming" / settings
   else:
     raise newException(ValueError, "Could not locate NWN root: unsupported OS")
-  let data = json.parseFile(settingsFile)
-  doAssert(data.hasKey("folders"))
-  doAssert(data["folders"].kind == JArray)
-
-  for release in releases:
-    for folder in data["folders"].getElems.mapIt(it.getStr / release):
-      if dirExists(folder / "data"):
-        info("Located", "Beamdog installation at " & folder)
-        return folder
+  if fileExists(settingsFile):
```

Comment:
This section is covered by another PR, but I don't think it will matter if we merge the other and then rebase this onto master.

Reply:
I made compilation comfirmation first, then had to work this because it wouldn't work with that file change.  I'm not very organized tonight, apparently.